Reflection_Engine/Query/MethodList.cs

```diff
@@ -59,6 +59,18 @@ public static List<MethodBase> AllMethodList()
             return m_AllMethodList;
         }
 
+        /***************************************************/
+
+        public static List<MethodBase> ExternalMethodList()
```

Comment:
This one collects the methods that come from external libraries.

I have chosen to store the methods into a standalone field to avoid contaminating the other ones. 
The reason is that the namespace of the methods the field stores do not start with `BH`.